Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/UwpPlayerBuildTools.cs

```diff
@@ -42,6 +44,91 @@ private static void ParseBuildCommandLine(ref UwpBuildInfo buildInfo)
             }
         }
 
+        private static void AddLiveCubeModelToProject(string filePath)
+        {
+            var text = File.ReadAllText(filePath);
+            var doc = new XmlDocument();
+            doc.LoadXml(text);
+            var root = doc.DocumentElement;
+
+            // Check to see if model has already been added
+            XmlNodeList nodes = root.SelectNodes("//None[@Include = \"Assets\\LiveCubeModel.glb\"]");
+            if (nodes.Count > 0)
+            {
+                Debug.Log("Model Already Exists in Project File");
+                return;
+            }
+
+            var newNodeDoc = new XmlDocument();
+            newNodeDoc.LoadXml("<None Include=\"Assets\\LiveCubeModel.glb\">" +
+                            "<DeploymentContent>true</DeploymentContent>" +
+                            "</None>");
+            var newNode = doc.ImportNode(newNodeDoc.DocumentElement, true);
+            var list = doc.GetElementsByTagName("ItemGroup");
+            var items = list.Item(1);
+            items.AppendChild(newNode);
+            doc.Save(filePath);
+        }
+
+        private static void AddLiveCubeModelToFilter(string filePath)
+        {
+            var text = File.ReadAllText(filePath);
```

Comment:
I feel like these all should exeucte just on a string in memory. Instead of reading/writing to disk every execution.

Whoever, is calling each of these can then read/write to disk i.e